sqlglot/expressions.py

```diff
@@ -3619,6 +3621,14 @@ def output_name(self) -> str:
         return self.name
 
 
+class Increment(Expression):
+    arg_types = {"this": True}
+
+
+class Start(Expression):
+    arg_types = {"this": True}
+
```

Comment:
We don't need these, right? You can just store the expressions as is in the corresponding arguments.

Btw, you could also do `pass` here.